src/meta.js

```diff
@@ -2,12 +2,11 @@
 
 import { base64encode } from 'belter/src';
 
-import { getScriptUrl, getStageHost, getAPIStageHost } from './script';
+import { getScriptUrl, getSDKAttributes } from './script';
 
 export function getSDKMeta() : string {
-    return base64encode(JSON.stringify({
-        url:          getScriptUrl(),
-        stageHost:    getStageHost(),
-        apiStageHost: getAPIStageHost()
-    })).replace(/\=+$/, ''); // eslint-disable-line no-useless-escape
+    const attrs = getSDKAttributes();
+
+    attrs.url = getScriptUrl();
+    return base64encode(JSON.stringify(attrs)).replace(/\=+$/, ''); // eslint-disable-line no-useless-escape
```

Comment:
Can we filter just `data-` attributes here?

Reply:
Wait, ignore this comment. `getSDKAttributes` is already doing this.

Reply:
https://github.com/paypal/paypal-sdk-client/blob/master/src/script.js#L34
`getSDKAttributes()` only collects `data-` attributes.